Open_Science_New.html

```diff
@@ -438,28 +478,30 @@ <h2>Extensible and modular design</h2>
             <img src="images/EEG.png">
               </section>
 
-	       <section data-background="images/DQT_Define_new.png" data-background-size="100%">	
+	       <section data-background="images/DQT_Define_new.png" data-background-size="100%">
 	    <h2><font color="0f4078">Cross-Modal Querying</font></h2>
 	  </section>
-	  
+
 	  <section data-background="images/DQT_Results_new.png" data-background-size="100%">
 	    <h2><font color="0f4078">Real-Time Query Results</font></h2>
 	  </section>
-	    
-	    <section>
-              <a href="CBRAIN_indepth.html#/cbrain-indepth-destination" target="_blank"><h2>WHAT IS CBRAIN?</h2></a>
+        </section>
+
+        <section>
+            <section>
+              <a href="CBRAIN_indepth.html#/cbrain-indepth-destination" target="_blank"><h2>Introducing... CBRAIN!</h2></a>
 	      <table class="reveal">
 		<tr>
 		  <td width="50%" style="vertical-align: middle">
 		    <a href="https://www.frontiersin.org/articles/10.3389/fninf.2014.00054/full" target="_blank"><img src="images/CBRAIN_paper.png" width="90%"></a>
 		  </td>
-		  
+
 		  <td width="50%" style="vertical-align: middle;">
 		    <a href="https://mcin-cnim.ca/technology/cbrain/" target="_blank"><img src="images/cbrain-blue.png" style="border:0; border-radius:30px;"></a>
 		    <p><small><br>“..is an <font color="12a3d8">OPEN SOURCE</font> web-based collaborative <font color="12a3d8">High-Performance Computing (HPC)</font> platform for <font color="12a3d8">neuroimaging research</font>, connecting researchers to the <font color="12a3d8">tools</font> required to handle <font color="12a3d8">large datasets</font>.”</small></p>
 		  </td>
 
-	</tr></table>          
+	</tr></table>
 	    </section>
 
 	     <section>
```

Comment:
*NOTE* @knoll-alex see how this CBRAIN slide answers "Where are we" and "who uses us" ? 
It would be good to mirror this info progression roughly in the first few LORIS slides (for this, use the info in the linked google slides in my comment in that section)